### ISSUE TITLE
Add more detail to automated checks role

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#385]
**Describe the bug**
From user feedback, emphasize the role of automated checks in the tool: "that no automated tool can find even a majority of accessibility defects on a web page"

**To Reproduce**
1. Go to https://www.washington.edu/accesscomputing/AU/before.html
2. Run automated checks

OR
1. Go to https://accessibilityinsights.io/docs/en/web/getstarted/fastpass

**Expected behavior**
The Automated checks section of FastPass should clarify that it only finds some of the most common accessibility issues

## Are you willing to submit a PR?
Yes

## Did you search for similar existing issues?
Yes

**Comment by ferBonnin** [comment]
@LiLoDavis can you help us draft the language for this?

**Comment by LiLoDavis** [comment]
@ferBonnin How about this? The "Did you know?" part is a bit of a trick to get people to look at the text. 

Did you know?
Automated checks can identify some common accessibility problems, such as missing or invalid properties. Most accessibility problems can be detected only through manual testing.

**Comment by ferBonnin** [comment]
@LiLoDavis I like this! Do we want to mention that they can use Assessment too?

**Comment by LiLoDavis** [comment]
@ferBonnin Good idea. How about this?

Did you know?
Automated checks can detect some common accessibility problems, such as missing or invalid properties. However, most accessibility problems can be identified only through manual testing. The most reliable way to evaluate the accessibility of a website or web app is to complete an assessment.

**Comment by ferBonnin** [comment]
@LiLoDavis looks great. I would just modify to add 'compliance': 
Did you know?
Automated checks can detect some common accessibility problems, such as missing or invalid properties. However, most accessibility problems can be identified only through manual testing. The most reliable way to evaluate the accessibility compliance of a website or web app is to complete an assessment.

**Comment by LiLoDavis** [comment]
@ferBonnin I approve!

**Comment by dbjorge** [comment]
@ferBonnin @LiLoDavis can you clarify exactly where in the UI/docs you want this new text to show up?

**Comment by ferBonnin** [comment]
@dbjorge only in the UI (already added it in the docs). Will send you a screenshot

**Comment by dbjorge** [comment]
To clarify: it should render in the fastpass view only, in between the `<h1>{this.props.title}</h1>` and the `this.renderContent()` in the `IssuesTable` within the details view main content. Probably we'd do this by adding an optional `subTitle` to `IssuesAdHocVisualization.displayableData` and rendering it in `IssuesTable` iff present.

It should be styled as `$secondary-text`.

**Comment by dbjorge** [comment]
Fix is implemented in #632, but need to wait til Chelsea is back in the office to review continue
